transparentpath/io/_io.py

```diff
@@ -1,26 +1,29 @@
 import builtins
-from typing import IO, Union, Any
+import tempfile
 from pathlib import Path
-from ..gcsutils.transparentpath import TransparentPath
+from typing import IO, Union, Any
 
+from ..gcsutils.transparentpath import TransparentPath
 
 builtins_open = builtins.open
 
 
 def myopen(*args, **kwargs) -> IO:
-    """Method overloading builtins' 'open' method, allowing to open files on GCS using TransparentPath."""
+    """Method overloading builtins' 'open' method, allowing to open files on GCS and scaleway and by ssh using
+    TransparentPath. """
     if len(args) == 0:
         raise ValueError("open method needs arguments.")
     thefile = args[0]
-    if type(thefile) == str and "gs://" in thefile:
+    if type(thefile) == str and ("gs://" in thefile):
```

Comment:
mabye this condition should also handle ssh : I do not think doing `open("sftp://...")` actually works, you would here too need to use a transparentpath